src/Components/test/E2ETest/ServerRenderingTests/FormHandlingTests/FormWithParentBindingContextTest.cs

```diff
@@ -1378,8 +1377,9 @@ public void RadioButtonGetsResetAfterSubmittingEnhancedForm()
 
         Browser.Exists(By.Id("submit-button")).Click();
 
-        Assert.False(Browser.Exists(By.Id("checkbox")).Selected);
-        Assert.False(Browser.Exists(By.Id("radio-button")).Selected);
+        var timeout = TimeSpan.FromSeconds(1);
```

Comment:
Hmm, this is not the way I think.

```
        Assert.False(Browser.Exists(By.Id("checkbox")).Selected);
```

will already wait and retry. Alternatively, check for other methods in `Browser` which could be use, that already include waits and retries

Reply:
I didn't notice that the default one waits as well, it should be equally good, thanks.

Reply:
but.. does it? `private static void WaitAssertCore(IWebDriver driver, Action assertion, TimeSpan timeout = default)` where default `TimeSpan` is `TimeSpan.Zero`. Am I wrong?